starlite/config/logging.py

```diff
@@ -49,17 +52,55 @@ class LoggingConfig(BaseModel):
     """This will be the configuration for the root logger. Processing of the configuration will be as for any logger,
     except that the propagate setting will not be applicable."""
 
-    def configure(self) -> None:
+    def _enable_rich_logging(self, log_config_dict: Dict[str, Any]) -> Dict[str, Any]:
+        """Overwrite the default console log configuration with rich log
+        configuration.
+
+        Args:
+            log_config_dict (Dict[str,Any]): dict representation of LoggingConfig model
+        Returns:
+            Dict[str,Any]: dict representation of LoggingConfig model with rich console logging in place of default console logging
+        """
+        log_config_dict["formatters"]["rich"] = {"format": "%(name)s - %(message)s"}
```

Comment:
Probably best to think of how the user can configure this.

Reply:
I agree that having the default format strings buried in the LoggingConfig model isn't great.  Would it work to put them in starlite.constants and import them?  Or is that too far removed from the logging config?

Reply:
constants should be placed at the top of the file probably, but this is not what im talking about.

The user should be able to define the formatter string. Here you only have name and message, but there are other variables exposed by the logger. 

I would try to add this as a class attribute.

Reply:
An attribute of LoggingConfig or Starlite?

Reply:
Of `LoggingConfig`